docs/features/kubernetes/proxy.md

```diff
@@ -40,25 +46,32 @@ const kubernetesProxyEndpoint = `${kubernetesBaseUrl}/proxy`;
 await fetch(`${kubernetesProxyEndpoint}/api/v1/namespaces`, {
   method: 'GET',
   headers: {
-    'X-Kubernetes-Cluster': CLUSTER_NAME,
-    Authorization: `Bearer ${token}`,
+    'Backstage-Kubernetes-Cluster': CLUSTER_NAME,
+    'Backstage-Kubernetes-Authorization': `Bearer ${token}`,
+    Authorization: `Bearer ${userToken}`,
   },
 });
 ```
 
 ## How it works
 
 The proxy will interpret the
-[`X-Kubernetes-Cluster`
-header](https://backstage.io/docs/reference/plugin-kubernetes-backend.header_kubernetes_cluster)
-as the name of the cluster to target. This name will be compared to each cluster
+[`Backstage-Kubernetes-Cluster`](https://backstage.io/docs/reference/plugin-kubernetes-backend.header_kubernetes_cluster)
+header as the name of the cluster to target. This name will be compared to each cluster
 returned by all the configured [cluster
 locators](https://backstage.io/docs/features/kubernetes/configuration#clusterlocatormethods)
--- the first cluster whose [`name` field](https://backstage.io/docs/features/kubernetes/configuration#clustersname) matches
-the value in the header will be targeted.
 
-Then the request will be forwarded verbatim (but with the endpoint's base URL
-prefix stripped) to the cluster.
+- the first cluster whose [`name` field](https://backstage.io/docs/features/kubernetes/configuration#clustersname) matches
+  the value in the header will be targeted.
```

Comment:
Something funny happened here; the `--` was supposed to be a long dash, but here it seems to have changed to become a bullet list item

Reply:
Reverted back to a long dash instead of a bullet point.